### DISCUSSION TITLE
The validating log does not remain in the console.

### DISCUSSION THREAD

**Discussion by Keiku** [Lightning-AI/pytorch-lightning#8012] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
The validating log does not remain in the console. How can I print and display it?

The current `validation_step` is as follows.

```python
    def validation_step(self, batch, batch_nb):
        loss, accuracy = self.forward(batch)
        self.log("loss/val", loss)
        self.log("acc/val", accuracy)
        return loss
```

It will be displayed for a moment as shown below, but it will not remain as a log.

```bash
Global seed set to 0
GPU available: True, used: True
TPU available: None, using: 0 TPU cores
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Files already downloaded and verified
Files already downloaded and verified
Epoch 26:  98%|███████████████▋| 114/116 [00:25<00:00,  4.50it/s, loss=0.623, v_num=0]
Validating: 100%|█████████████████████████████████████| 19/19 [00:01<00:00, 18.68it/s]
```

```
Global seed set to 0
GPU available: True, used: True
TPU available: None, using: 0 TPU cores
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Files already downloaded and verified
Files already downloaded and verified
Epoch 28:   3%|▌                 | 4/116 [00:01<00:42,  2.62it/s, loss=0.588, v_num=0]
```

**Comment by awaelchli** [comment, marked as answer]
Yes that's correct, it's a UI design choice. 
If you wish to see your values in the progress bar, you can log with `self.log("loss/val", loss, prog_bar=True)` and it will show up in the main bar.

**Reply by tchaton** [reply]
Hey @Keiku,

As shared by @awaelchli, this would print to the progress bar.
Learn more with this section of the doc: https://pytorch-lightning.readthedocs.io/en/latest/extensions/logging.html#logging-from-a-lightningmodule

Best,
T.C

**Reply by Keiku** [reply]
It wasn't the answer I expected, but I found that it didn't log by default. I will consider if I can implement a custom logger.
